Source/Core/DolphinQt/GameList/GameListModel.h

```diff
@@ -58,6 +58,7 @@ class GameListModel final : public QAbstractTableModel
     FileFormat,
     BlockSize,
     Compression,
+    TimePlayed,
     Tags,
     Count,
   };
```

Comment:
Is it safe to add this here? I feel like it would need to be added to the end to avoid shifting numeric values, but I'm not entirely sure how the column list is serialized.

Reply:
It shouldn't be an issue; the column accesses are written like Column::TimePlayed so it shouldn't matter how the ordering is done.

Reply:
The issue is that Qt remembers which columns the user has enabled or not, and that means it needs to get saved somewhere; if it's saving the numeric value associated with the enum, then that would cause problems (and I don't see anything that e.g. converts the enum to a string).

I guess the way to test this is enabling the tags column on master and then switching to your PR and seeing if tags stays enabled, or if it becomes time played.

Reply:
I just tested it out and it seems to work out fine; with tags disabled on both, enabling tags on the master enables tags on my PR, not the TimePlayed column.